.github/workflows/ios.yml

```diff
@@ -33,6 +33,15 @@ jobs:
       - name: Install cocoapods
         run: gem install xcpretty cocoapods
 
+      - name: TEMP manually run pod install
```

Comment:
Did you add this again on purpose?

Reply:
Yes, it's not ideal, but it's the only way I've found to get an updated Podfile.lock without using a mac.